rest/data/interface.go

```diff
@@ -73,11 +73,12 @@ type Connector interface {
 	// RestartBuild is a method to restart the build matching the same BuildId.
 	RestartBuild(string, string) error
 
-	// Find project variables matching given projectId. If bool is set, returns only redacted values.
+	// Find project variables matching given projectId. If bool is set, also return private variables.
 	FindProjectVarsById(string, bool) (*restModel.APIProjectVars, error)
-	// UpdateProjectVars updates the project using the variables given in the model.ggg
+	// UpdateProjectVars updates the project using the variables given
+	// in the model (removing old variables if overwrite is set).
 	// If successful, updates the given projectVars with the updated projectVars.
-	UpdateProjectVars(string, *restModel.APIProjectVars) error
+	UpdateProjectVars(string, *restModel.APIProjectVars, bool) error
```

Comment:
is there a way to make the boolean a field in the projectVars struct?

Reply:
I know you don't like booleans in function signatures :) but I would rather not. That's not actually where it belongs, it's just a janky fix. I could move copyVariableOptions into rest/model so that can be passed into routes instead?

Reply:
it's fine. I just want you to hate booleans in function signatures as much as I do, I think it's fine.